OpenEdXMobile/src/main/java/org/edx/mobile/module/prefs/PrefManager.java

```diff
@@ -266,6 +266,11 @@ public void setIsVideosCacheRestored(boolean restored) {
         public static final String AppSettingNeedSyncWithParse = "AppSettingNeedSyncWithParse";
         public static final String UserPrefVideoModel = "UserPrefVideoModel";
         public static final String LAST_COURSE_STRUCTURE_FETCH = "LastCourseStructureFetch";
+        /**
+         * For downloaded videos to appear in order on the My Videos screen, we need
```

Comment:
"in order" is the whole reason why we need the videos' courses data cached. Cuz the order in which the videos have been downloaded may differ from the order they appear in courseware.